processes/credential_exposure.md

```diff
@@ -0,0 +1,53 @@
+# Credential exposure
+
+Credentials are things like passwords or API keys. They are any values you wouldn't want exposed to the public but which your app or service needs to do its job.
+
+We haven't settled on a specific standard for how we handle them e.g. [12 factor app](https://12factor.net/config). But the one thing we are all agreed on is that they should not be committed with your project's source code.
+
+Yet mistakes happen and sometimes these secrets become exposed.
+
+This process will take you through what to do if that happens. The key point is
+
+> once out *always* consider your secret as compromised
+
+No matter the speed with which you rectify the situation, or how little you think the chance is for people to see it. Treat it as compromised.
+
+## Contact web-ops
```

Comment:
This is correct but is specific to AWS credentials. The main thing is to get the credential revoked (whatever it is) so I think it would be worth making this point generically and then covering the specifics for AWS.